adagrams/game.py

```diff
@@ -1,11 +1,129 @@
+import random 
+
+
+# random allows us to accept 
+# random letters from LETTER_POOL 
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
+score_chart = {
+    'A': 1,
+    'B': 3,
+    'C': 3,
+    'D': 2,
+    'E': 1,
+    'F': 4,
+    'G': 2,
+    'H': 4,
+    'I': 1,
+    'J': 8,
+    'K': 5,
+    'L': 1,
+    'M': 3,
+    'N': 1,
+    'O': 1,
+    'P': 3,
+    'Q': 10,
+    'R': 1,
+    'S': 1,
+    'T': 1,
+    'U': 1,
+    'V': 4,
+    'W': 4,
+    'X': 8,
+    'Y': 4,
+    'Z': 10
+  }
+
 def draw_letters():
-    pass
+    players_hand = []
+    
+    for letter in LETTER_POOL.keys():
+        players_hand.append(letter)
+    
+    random.shuffle(players_hand)
```

Comment:
Nice use of a built-in function! How does shuffle work under the hood?

I encourage students to practice writing this kind of logic on their own because it provides practice in thinking through the "how" part while coding. While writing it out on your own is more lines of code, I think that's ok for initial projects to you get this practice.

Especially in interviews, we should feel confident that we understand how built-in functions work under the hood because your interviewer may ask you to explain it to them.